string color_name
string type
geometry_msgs/Vector3 player_distance
geometry_msgs/Vector3 half_size